components/clp-package-utils/clp_package_utils/scripts/archive_manager.py

```
@@ -158,6 +165,8 @@ def main(argv: typing.List[str]) -> int:
         logger.exception("Failed to load config.")
         return -1
 
+    dataset = validate_dataset(clp_config, parsed_args.dataset)
```

Flake8 (7.2.0)

[error] local variable 'dataset' is assigned to but never used

(F841)